backend/app/routers/authentication/authenticate.py

```
@@ -0,0 +1,237 @@
+    get_user_by_field,
+    get_user,
+    update_user
+)
+import re
+from fastapi import HTTPException
+from datetime import datetime
+import bcrypt
+import random
+
+import jwt
+from datetime import datetime, timedelta, timezone
```

Ruff (0.8.0)

Redefinition of unused `datetime` from line 12

Remove definition: `datetime`

(F811)